build/lib/packager.js

```diff
@@ -152,8 +113,9 @@ class Packager {
 			this.generateManifestJSON(),
 			// copy misc dirs/files over
 			this.copy([ 'CREDITS', 'README.md', 'package.json', 'cli', 'templates' ]),
-			// transpile/bundle and copy common/ JS files
-			this.transpile(),
+			// copy over 'common' bundle
+			fs.ensureDir(path.join(this.zipSDKDir, 'common', 'Resources')),
```

Comment:
These two won't run in series, these are all things run in parallel! while this is similar to the `async` library usage we do in the CLI, I'm gathering the all up as Promises to run in parallel via `Promise.all`. If you have specific code that needs to run in series, extract it out to an async function you can call here instead.